diff --git a/train_async.py b/train_async.py
index 94cc2969..a6b65961 100644
--- a/train_async.py
+++ b/train_async.py
@@ -4,6 +4,9 @@ from slime.ray.placement_group import create_placement_groups, create_rollout_ma
 from slime.utils.arguments import parse_args
 from slime.utils.logging_utils import configure_logger, finish_tracking, init_tracking, update_tracking_open_metrics
 from slime.utils.misc import should_run_periodic_action
+from slime.utils.common import is_npu
+if is_npu():
+    import mindspeed.megatron_adaptor
 
 
 # The framework supports other asynchronous approaches such as fully async (which is shown in examples/full_async).